core/export.ts

```diff
@@ -0,0 +1,158 @@
+import { loadValue } from '@logux/client'
+import { atom, onMount } from 'nanostores'
+
+import {
+  type CategoryValue,
+  feedsByCategory,
+  getCategories,
+  getCategoryTitle
+} from './category.js'
+import { type FeedValue, getFeeds } from './feed.js'
+import { readonlyExport } from './lib/stores.js'
+import { getPosts, type PostValue } from './post.js'
+
+type ExtendedFeedValue = {
+  posts?: PostValue[]
+} & FeedValue
+
+let $categories = atom<CategoryValue[]>([])
+let $allFeeds = atom<FeedValue[]>([])
+let $feedsByCategoryList = atom<[CategoryValue, ExtendedFeedValue[]][]>([])
+let $creating = atom<boolean>(false)
+
+export const feedsByCategoryList = readonlyExport($feedsByCategoryList)
+export const creating = readonlyExport($creating)
+
+onMount(feedsByCategoryList, () => {
+  Promise.all([loadValue(getCategories()), loadValue(getFeeds())]).then(
+    ([categoriesValue, feedsValue]) => {
+      $categories.set(categoriesValue.list)
+      $allFeeds.set(feedsValue.list)
+      $feedsByCategoryList.set(
+        feedsByCategory($categories.get(), $allFeeds.get())
+      )
+
+      selectAllExportedFeeds()
```

Comment:
Seems like we need to mark that `feedsByCategoryList` is not just regular list, but special for export.

For instance, by renaming `feedsByCategoryList`.